docs/source/build-from-source.mdx

```diff
@@ -1,13 +1,13 @@
 ---
 title: Building Apollo Router from source
-sidebar_title: 🛠 Build from source
+sidebar_title: Build from source
```

Comment:
Now that the article count is growing, probably easiest to do away with the emoji